### PR TITLE
Log operator config on startup

### PR DESCRIPTION
**Pull request by glaksh100**

### CODE REVIEW

File: cmd/flinkk8soperator/cmd/root.go [hunk 1]
```diff
@@ -140,6 +140,19 @@ func executeRootCmd(controllerCfg *controller_config.Config) error {
 	}
 }
 
+func logConfigMap(ctx context.Context, controllerCfg *controller_config.Config) {
+	logger.Infof(ctx, "Staleness Duration %v", controllerCfg.StatemachineStalenessDuration)
```
Comment (mwylde) [inline cmd/flinkk8soperator/cmd/root.go]: Is it possible to use reflection to do this dynamically? That might lessen the work of keeping this in sync with the config as it evolves.

Comment (anandswaminathan) [reply cmd/flinkk8soperator/cmd/root.go]: +1

I believe you can simply do `logger.Infof(ctx, "%+v", controllerCfg)`

Comment (glaksh100) [reply cmd/flinkk8soperator/cmd/root.go]: Thanks y'all. Updated to use `logger.Infof(ctx, "%+v\n", controllerCfg)`

**Comment by glaksh100** [comment]
🚀

**Pull request merged**